docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.5'
+
+services:
+  frontend:
+    image: 'jbromfield/t1-frontend:v1'
```

Comment:
is this useful for the devs, who want to build the version of the files they have locally?

Reply:
removed docker files from this feature branch.